src/pages/MyTokens/NFTPage.tsx

```diff
@@ -139,7 +142,7 @@ export const NFTPage = () => {
       }
       : null
     );
-  }, [myCollections, isFetchingTokens]);
```

Comment:
I belive this one was here to reset filters when some options are changed (ex. collection or my_tokens)